libs/state/package.json

```
@@ -48,8 +48,8 @@
     "url": "https://github.com/rx-angular/rx-angular.git"
   },
   "peerDependencies": {
-    "@angular/core": ">=12.0.0",
-    "@angular/platform-browser": ">=12.0.0",
+    "@angular/core": ">=13.0.0",
+    "@angular/platform-browser": ">=13.0.0",
```

issue: Only mark Angular 13.x as officially supported
```suggestion
    "@angular/core": "^13.0.0",
    "@angular/platform-browser": "^13.0.0",
```